### ISSUE TITLE
Add "wcaid" phrase

### ISSUE THREAD

**Issue by leduyquang753** [thewca/tnoodle#301]
Hi again! Today I'm here to post another small suggestion. This time, it's about the "**WCA ID**" phrase in the FMC sheet.
In some languages, the noun phrases are inverted comparing to English. In this case, in my Vietnamese language for example, "**WCA ID**" is "**ID WCA**", and the FMC sheet should look like:
![sample](https://user-images.githubusercontent.com/26449060/34776654-31bc4da4-f64a-11e7-95c6-1e52e096deb9.png)
(I don't know if you noticed or not, but I changed the font to **Segoe UI**.)

**TL; DR:** I suggest you to make a new small phrase called **wcaid**.
Thanks.

**Comment by jfly** [comment]
Please feel free to send in a PR implementing this. We won't include it in the next release of tnoodle, but maybe the one after that. Thanks!

**Comment by jfly** [comment]
@leduyquang753, why did you close this issue? You're still working on this over in #302, right?

**Comment by leduyquang753** [comment]
I thought there should be only one place to work with? Because this is a request and the PR is my actual work? :confused:

**Comment by jfly** [comment]
The preferred behavior is to leave an issue open until the code that fixes it has actually be merged to master. If you close the issue before the code is merged to master, it's still possible that the PR never gets merged and then the issue never actually gets fixed.

Please read through https://help.github.com/articles/closing-issues-using-keywords/, which shows a very easy way of implementing this behavior.

**Comment by leduyquang753** [comment]
Oh it blows my mind. Thanks. :sweat_smile:
